app/views/pages/_header.html.erb

```diff
@@ -21,7 +21,12 @@
                 <li class="nav-item">
                     <%= link_to "Admin Dashboard", admin_root_path, class:"nav-link"%>
                 </li>
+                <% if admin_user_signed_in? %>
+                <li class="nav-item">
+                    <%= link_to "Log Out Admin", destroy_admin_user_session_path, class:"nav-link"%>
+                </li>
+                <% end %>
```

Comment:
This starts to look badly - you are starting to have a lot of ifs in the view.

Consider using separate layout for admin pages, wit separate navbar

Reply:
I would consider giving an example and whys it's helpful instead of judging

Reply:
@ciejj explained it to me on 1on1 call :)